python/tvm/relay/op/transform.py

```diff
@@ -139,3 +139,40 @@ def take(data, indices, axis=None):
         The computed result.
     """
     return _make.take(data, indices, axis)
+
+
+def where(condition, x, y):
+    """Selecting elements from either x or y depending on the value of the
+    condition.
+
+    Parameters
+    ----------
+    condition : relay.Expr
+        The condition array.
+
+    x : relay.Expr
+        The first array to be selected.
+
+    y : relay.Expr
+        The second array to be selected.
+
+    Returns
+    -------
+    result : relay.Expr
+		The selected array.
+
+    Examples
+    --------
+    .. code-block:: python
+
+        x = [[1, 2], [3, 4]]
+        y = [[5, 6], [7, 8]]
+        condition = [[0, 1], [-1, 0]]
+        where(conditon, x, y) = [[5, 2], [3, 8]]
+
+        condition = [1, 0]
+        where(conditon, x, y) = [[5, 2], [3, 8]]
```

Comment:
is it true?

Reply:
@yzhliu oops. My bad, forgot to update the values.